core/ledger/kvledger/txmgmt/statedb/statecouchdb/statecouchdb.go

```diff
@@ -16,7 +16,6 @@ import (
 	"github.com/hyperledger/fabric/common/flogging"
 	"github.com/hyperledger/fabric/common/ledger/dataformat"
 	"github.com/hyperledger/fabric/common/metrics"
-	"github.com/hyperledger/fabric/core/common/ccprovider"
```

Comment:
Very nice ... we need to do more of this!

Reply:
Thanks, @mastersingh24 If you keep on encouraging me like this, many more will come. Anyway, we do have some time limit to complete these tasks before starting the checkpoint pruning. Will do as much as @manish-sethi and I can do to improve the code before that.